packages/workflow-runtime/internal/activities.ts

```diff
@@ -72,6 +78,25 @@ export const createActivities = () => ({
       throw serializeAsTemporalError(err);
     }
   },
+
+  createSleepActivity: async (d: SleepDuration, opts: ClientOptions = {}): Promise<void> => {
+    const client = new Client(opts);
+    try {
+      // Use the time the activity was scheduled as the timer start time
```

Comment:
Let's dump some context as a comment here. Thanks for checking w/ them in Slack!

```suggestion
      // Compute when the sleep should end based on the timestamp of when this activity
      // was originally scheduled (IOW, if Temporal performs multiple attempts at executing this
      // activity, this timestamp will always be the same).
```